rpm/run_tests_on_vms.sh

```diff
@@ -52,11 +60,15 @@ for IMAGE in "${IMAGES[@]}"; do
 	echo "Creating a VM with \"$IMAGE\""
 	NAME="voyager"
 	openstack server delete --wait $NAME 2>/dev/null # just to be sure
-	openstack server create --image "$IMAGE" --flavor m1.small \
-		--key-name jenkins_key --nic net-id=dualStack --wait $NAME \
-		> /dev/null
-	IP=$(openstack server list | grep $NAME | \
-		grep -oE '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}')
+	IP=""
+	until [[ $IP != "" ]]; do
```

Comment:
UH-IaaS has fixed the network problem, perhaps the loop should be removed now.
https://github.com/norcams/himlar/commit/7f21da53979ef458c2f23343faf0fcf3f0a401bc